html.go

```diff
@@ -483,9 +488,15 @@ func (options *Html) AutoLink(out *bytes.Buffer, link []byte, kind int) {
 }
 
 func (options *Html) CodeSpan(out *bytes.Buffer, text []byte) {
-	out.WriteString("<code>")
-	attrEscape(out, text)
-	out.WriteString("</code>")
+	if options.flags&HTML_TOC_MD == 0 {
+		out.WriteString("<code>")
+		attrEscape(out, text)
+		out.WriteString("</code>")
+	} else {
+		out.WriteString("`")
+		out.Write(text)
+		out.WriteString("`")
+	}
```

Comment:
I don't like the extra complexity this brings, but I don't see a different way of doing this if we're to support this feature.